### PR TITLE
grow cluster test: support for custom node number

### PR DESCRIPTION
**Pull request by larisau**
for testing only, not merge

### CODE REVIEW

**Comment by amoskong** [comment]
Hi @larisau , is it a real PR? Let's close it if you don't want it to be merged.

**Comment by roydahan** [comment]
We can leave it for now.
We haven't decided what do to with it.

**Comment by roydahan** [comment]
ready for review.

File: sdcm/cluster.py [hunk 1]
```diff
@@ -3138,16 +3143,16 @@ def node_setup(node):
             if self._experimental():
                 self._node_setup(node=node)
             node.wait_db_up(verbose=verbose)
-            node.remoter.run('sudo yum install -y scylla-gdb',
-                             verbose=verbose, ignore_status=True)
+            # node.remoter.run('sudo yum install -y scylla-gdb',
+            #                  verbose=verbose, ignore_status=True)
             queue.put(node)
```
Comment (roydahan) [inline sdcm/cluster.py]: @larisau , we do not want to remove this from all of our testing machines.
Maybe for this test it's ok, but not for others.

File: sdcm/cluster.py [hunk 2]
```diff
@@ -3138,16 +3143,16 @@ def node_setup(node):
...
             queue.task_done()
 
         start_time = time.time()
 
         for node in node_list:
             node.wait_ssh_up(verbose=verbose)
-            self.collectd_setup.install(node)
+            # self.collectd_setup.install(node)
```
Comment (roydahan) [inline sdcm/cluster.py]: Same as above, unless we don't need the collectd anymore... 
@amoskong, do you know if we still need it?

File: tests/aws-grow-cluster-200nodes.yaml [hunk 3]
```diff
@@ -0,0 +1,61 @@
...
+            subnet_id: 'subnet-d934e980'
+            ami_id_db_scylla: 'ami-7e137d68'
+            ami_db_scylla_user: 'centos'
+            ami_id_loader: 'ami-7e137d68'
+            ami_loader_user: 'centos'
+            ami_id_db_cassandra: 'ami-3eff0028'
+            ami_db_cassandra_user: 'ubuntu'
+            ami_id_monitor: 'ami-3d101f2a'
+            ami_monitor_user: 'centos'
+    gce: !mux
```
Comment (roydahan) [inline tests/aws-grow-cluster-200nodes.yaml]: Does this test work also in gce?
Your yaml file called aws-... but you have a mux also for gce.

IIRC, it took very long time to run it in gce due to all installations we are doing per node.

**Comment by larisau** [comment]
now it's for merge, please review

**Pull request merged**